.circleci/config.yml

```diff
@@ -46,15 +46,32 @@ jobs:
           key: klaatu-lint-{{ checksum "poetry.lock" }}
           paths:
             - .tox
-  test-nightly:
+  test-nightly-firefox:
     working_directory: ~/klaatu
     machine:
       image: ubuntu-2204:2024.01.1
     steps:
       - checkout
       - run:
           name: Build docker image
-          command: docker build -t klaatu:latest -f Dockerfile.desktop .
+          command: docker build -t klaatu:firefox-nightly -f Dockerfile.desktop --build-arg="FIREFOX_VERSION=-nightly" .
+      - run:
+          name: Run tests
+          command: |
+            set -ex
+            docker compose up -d
+            docker compose run klaatu tox -e bdd-tests -- --reruns 2 --experiment-branch control --variables tests/fixtures/test_experiment.json --run-update-test --private-browsing-enabled
```

Comment:
Q: What does `--reruns 2` do?
Is that saying run the test suite twice, or it can fail 1-2&times; and it will retry?

Reply:
Yeah, on CI it's weird and likes to fail but doesn't fail locally. It's something to do with the ```simplehttpserver``` plugin im using, I need to open a patch on it